dashboards-observability/public/components/common/live_tail/__tests__/live_tail_button.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { configure, mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { LiveTailButton, StopLiveButton } from '../live_tail_button';
+import { waitFor } from '@testing-library/dom';
+
+  describe('Live tail button', () => {
```

Comment:
I believe the `describe` should not be indented, can we reduce the indent here?

Reply:
yeah okay, one more PR is yet to come, will include the change in that